### PR TITLE
slight reword of the verification section

### LINKED ISSUE #18: more precision in the verification section
* instead of "CWT" we should reference the section that describes the verification requirements
* InstanceID should be clearly marked as the lookup handle for the verification key material

### PR DESCRIPTION
**Pull request by thomas-fossati**
* add ref to Section 5 re: cryptographic verification requirements;
* mark Instance ID as the primary lookup handle for the verification key
  material.

Also s/verifier/Verifier/g

Fixes #18

### CODE REVIEW

File: draft-tschofenig-rats-psa-token.md [hunk 1]
```diff
@@ -531,30 +532,33 @@ keys.
...
   original baseline release. A verification policy may then allow this value to
   match any point on that release sequence or expect some minimum level of
   maturity related to the sequence.
 * Software Component, Signer ID - where present in a deployment, this could
-  allow a verifier to operate a more general policy than that for Measurement
+  allow a Verifier to operate a more general policy than that for Measurement
   Value as above, by allowing a token to contain any firmware entries signed by
   a known Signer ID, without checking for a uniquely registered version.
+* Certification Reference - if present, this value could be used as a hint to
+  locate an entry in the PSA Certification database associated with the
```
Comment (adrianlshaw) [inline draft-tschofenig-rats-psa-token.md:560]: There may or may not be a single database or certification scheme. To address this, might be better to rephrase "locate security certification information associated with the attesting device. An example could be a certificate number issued by www.psacertified.org" 

What do you think?

Comment (thomas-fossati) [reply draft-tschofenig-rats-psa-token.md]: thanks!  Your suggested text is in b45adbe

**Review by adrianlshaw — APPROVED** [review]
Nice edits. LGTM

**Review by SimonFrost-Arm — APPROVED** [review]
LGTM

File: draft-tschofenig-rats-psa-token.md [hunk 2]
```diff
@@ -531,30 +538,33 @@ keys.
 # Verification
 
 To verify the token, the primary need is to check correct formation and signing
```
Comment (hannestschofenig) [inline draft-tschofenig-rats-psa-token.md:540]:
```suggestion
To verify the token, the primary need is to check correct formatting and signing
```

File: draft-tschofenig-rats-psa-token.md [hunk 3]
```diff
@@ -531,30 +538,33 @@ keys.
...
-policy may require that the relevant claims must have a match to a registered
-reference value.  All claims may be worthy of additional appraisal.  It is
-likely that most deployments would include a policy with appraisal for the
-following claims:
-
-* Instance ID - the value of the Instance ID can be used (together with the kid
-  in the token COSE header, if present) to assist in locating the public key
-  used to verify the token signature.
+as detailed in {{sec-token-encoding-and-signing}}.  In particular, the Instance
+ID claim is used (together with the kid in the token COSE header, if present)
```
Comment (hannestschofenig) [inline draft-tschofenig-rats-psa-token.md:542]:
```suggestion
ID claim is used (together with the kid in COSE header, if present, protecting the CWT token)
```

File: draft-tschofenig-rats-psa-token.md [hunk 4]
```diff
@@ -531,30 +538,33 @@ keys.
...
+to assist in locating the public key used to verify the token signature.
```
Comment (hannestschofenig) [inline draft-tschofenig-rats-psa-token.md:543]:
```suggestion
to assist in locating the public key used to verify the signature covering the CWT token.
```

**Comment by hannestschofenig** [comment]
Looks good to me. Just minor editorial changes.

**Pull request merged**